sls-versions/src/main/java/com/palantir/sls/versions/SlsVersionType.java

```diff
@@ -45,6 +45,10 @@ Parser getParser() {
         this.priority = priority;
     }
 
+    /**
+     * Provides a first-level ordering of versions that have the same major/minor/patch versions. Version types with a
+     * higher priority are greater than version types with a lower priority.
```

Comment:
Thoughts on making it clear in the docs that there exists a case of two types sharing the same priority?
```suggestion
     * Provides a first-level ordering of versions that have the same major/minor/patch versions. Version types with a
     * higher priority are greater than version types with a lower priority.
     * Note that RELEASE_CANDIDATE and RELEASE_CANDIDATE_SNAPSHOT have the same priority.
```